protocol/contracts/amo/helpers/PoolHelper.sol

```diff
@@ -0,0 +1,248 @@
+pragma solidity ^0.8.4;
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "../interfaces/AMO__IBalancerVault.sol";
+import "../helpers/AMOCommon.sol";
+
+
+interface IWeightPool2Tokens {
+    function getNormalizedWeights() external view returns (uint256[] memory);
+}
+
+contract PoolHelper is Ownable {
```

Comment:
Out of interest, do you see any benefits of an abstract base contract rather than using a library?
Much of a muchness, just a style preference?

Reply:
exceeded the contract byte size limit. So moved this from former library(also tried abstract contract) implementation to a separate contract